README.rst

```diff
@@ -4,53 +4,114 @@ TruFont
 =======
 
 `TruFont <https://trufont.github.io>`__ is a font-editing application
-written with Python3, ufoLib, defcon and PyQt5.
+written with Python3, ufoLib, defcon, and PyQt5.
+
+.. image:: Lib/trufont/resources/screenshot.png
 
 Getting started
-~~~~~~~~~~~~~~~
+===============
+
+Quick installation instructions
```

Comment:
I would insert the `pip install trufont` instruction here, with an explanation of its limitation (depending on the developer team making release packages.)